File: api/src/Entity/File.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\HttpFoundation\Tests\StringableObject;

/**
 * @ORM\Entity()
 * @ORM\Table(name="files")
 */
class File
{
    /**
     * @ORM\Id
     * @ORM\Column(type="string")
     * @ORM\GeneratedValue(strategy="UUID")
     */
    protected $file_id;

    /**
     * @ORM\Column(type="string")
     */
    protected $file_name;

    /**
     * @ORM\Column(type="string")
     */
    protected $file_path;

    /**
     * @ORM\Column(type="string")
     */
    protected $file_type;


    public function getFileId()
    {
        return $this->file_id;
    }

    public function setFileId($id)
    {
        $this->file_id = $id;
        return $this;
    }

    public function getFileName()
    {
        return $this->file_name;
    }

    public function setFileName($name)
    {
        $this->file_name = $name;
        return $this;
    }

    public function getFilePath()
    {
        return $this->file_path;
    }

    public function setFilePath($path)
    {
        $this->file_path = $path;
        return $this;
    }

    public function getFileType()
    {
        return $this->file_type;
    }

    public function setFileType($type)
    {
        $this->file_type = $type;
        return $this;
    }

}

File: openex-api/src/main/java/io/openex/database/model/File.java
package io.openex.database.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import org.hibernate.annotations.GenericGenerator;

import javax.persistence.*;

@Entity
@Table(name = "files")
public class File implements Base {
    @Id
    @Column(name = "file_id")
    @GeneratedValue(generator = "UUID")
    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
    @JsonProperty("file_id")
    private String id;

    @Column(name = "file_name")
    @JsonProperty("file_name")
    private String name;

    @Column(name = "file_path")
    @JsonProperty("file_path")
    private String path;

    @Column(name = "file_type")
    @JsonProperty("file_type")
    private String type;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}
